content/BindingMethods/BindingMethods.facts.md

```diff
@@ -1,10 +1,10 @@
 ### Facts for Prepared Statement with Binding Methods
 
-Parameter binding in a prepared statement is essential against SQL injections. The `bindValue()` and `bindParam()` methods bind variables or values in a prepared statement that contains placeholders as parameters such as a positional placeholder `?` or named placeholder `:variable_name`.
+Parameter binding is useful to set the proper data type explicitly. The `bindValue()` and `bindParam()` methods bind variables or values in a prepared statement that contains placeholders as parameters such as a positional placeholder `?` or named placeholder `:variable_name`.
 
-- The `PDOStatement::bindValue()` method binds a value or variable to a parameter. It passes both `value` and `variable`. 
+- The `PDOStatement::bindValue()` method binds a value to a parameter.
 
-- The `PDOStatement::bindParam()` method binds a parameter exclusively to a specified variable. The variable is bound as a reference.
+- The `PDOStatement::bindParam()` method binds a parameter to a variable. The variable is bound as a reference and will be evaluated once the `PDOStatement::execute()` is called.
```

Comment:
I just need to clarify this statement:

> and will be evaluated once the `PDOStatement::execute()` is called.

Does this mean that `bindParam()` is evaluated only when the `PDOStatement::execute()` is called? Or `bindParam()` is evaluated only once when  the `PDOStatement::execute()` is called? Please check. Thanks.